### PR TITLE
Removing Erik code

### PR DESCRIPTION
**Pull request by nicolas-chaulet**
Some more cleanup

- Remove Erik's code for convs and sequential models
- Simplify API by removing the call to the set_input method

### CODE REVIEW

File: torch_points3d/core/base_conv/dense.py [hunk 1]
```diff
@@ -138,9 +137,9 @@ def __repr__(self):
 
 
 class GlobalDenseBaseModule(torch.nn.Module):
-    def __init__(self, nn, aggr="max", bn=True, activation="LeakyReLU", **kwargs):
+    def __init__(self, nn, aggr="max", bn=True, activation=torch.nn.LeakyReLU(negative_slope=0.1), **kwargs):
         super(GlobalDenseBaseModule, self).__init__()
-        self.nn = pt_utils.SharedMLP(nn, bn=bn, activation=get_activation(activation))
+        self.nn = MLP2D(nn, bn=bn, activation=activation, bias=False)
```
Comment (tchaton) [inline torch_points3d/core/base_conv/dense.py:142]: Why removing get_activation ? It was doing automatic resolution.

File: torch_points3d/core/common_modules/base_modules.py [hunk 2]
```diff
@@ -132,11 +128,32 @@ def forward(self, features, category_labels, **kwargs):
...
     def __init__(self, num_features, momentum=0.1):
         super().__init__()
-        self.batch_norm = BN(num_features, momentum=momentum)
+        self.batch_norm = nn.BatchNorm1d(num_features, momentum=momentum)
 
-    def forward(self, x):
+    def _forward_dense(self, x):
+        return self.batch_norm(x)
+
+    def _forward_sparse(self, x):
```
Comment (tchaton) [inline torch_points3d/core/common_modules/base_modules.py:136]: Small explanation here will be great for other people.

Comment (nicolas-chaulet) [reply torch_points3d/core/common_modules/base_modules.py:136]: Done

File: torch_points3d/core/base_conv/dense.py (hunk 1 above)
Comment (nicolas-chaulet) [reply torch_points3d/core/base_conv/dense.py:142]: We were not using it. You can still change the activation by giving the full torch module

**Pull request merged**